### PR TITLE
Add's Seven new Recipes for Chefs

### PR DESCRIPTION
**Pull request by Arthurdentist**
Can now cook your own chips. This isn't finished yet, I need to go over the values as far as nutriment and bitesize goes.

chips2

### CODE REVIEW

**Comment by dylanstrategie** [comment]
Oh god I was working on food items

Why would you do this

**Comment by Arthurdentist** [comment]
this is the first time I have heard you were working on food items at all.

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 1]
```diff
@@ -4467,3 +4474,18 @@
...
+	name = "Honk Serum"
+	id = "honkserum"
+	description = "Concentrated honking"
+	reagent_state = LIQUID
+	color = "#F2C900" // rgb: 242, 201, 0
+	custom_metabolism = 0.01
+
+/datum/reagent/honkserum/on_mob_life(var/mob/living/M)
+	if(prob(1))
+		if(prob(90))
```
Comment (ComicIronic) [inline code/modules/reagents/Chemistry-Reagents.dm]: But why?
If prob(0.9)

**Comment by Boxe** [comment]
>add's
>'s

**Comment by Arthurdentist** [comment]
go away boxed

**Comment by Arthurdentist** [comment]
bloody merge conflicts, can't be dealing with this.

**Pull request closed without merging**